store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9935,4 +9935,12 @@ TODO: delete them permanently from here
 <attr id="4006" name="zimbraServerExternalStoreConfig" type="string" cardinality="single" optionalIn="server" since="9.1.0">
   <desc>Server level external store config</desc>
 </attr>
+<attr id="4007" name="zimbraSMMultiReaderEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited"  since="9.1.0">
```

Comment:
4007, 4008 are already used by zimbraGlobalBackupEnabled and zimbraBackupSyncUsersEnabled. 4005 is used by zimbraActiveSyncVersion. Can you please check the attribute ids in develop, feature/9.1.0-Backup-Restore branches and other active PRs. Some PRs are already approved and merged, so we should be updating this one accordingly.

Reply:
revisited the numbers and rebased the branch